vochain/scrutinizer/process.go

```diff
@@ -114,20 +134,34 @@ func (s *Scrutinizer) ProcessList(entityID []byte, namespace uint32,
 }
 
 // ProcessCount return the number of processes indexed
-func (s *Scrutinizer) ProcessCount() int64 {
-	c, err := s.db.Count(&Process{}, nil)
+func (s *Scrutinizer) ProcessCount(entityID []byte) int64 {
+	var c int
+	var err error
+	if len(entityID) == 0 {
+		c, err = s.db.Count(&Process{}, nil)
+	} else {
+		c, err = s.db.Count(&Process{}, badgerhold.Where("EntityID").Eq(entityID))
```

Comment:
Index by EntityID

Reply:
Add a TO-DO if you want for Caching also the Process count by EntityID (we already doing it with envelopes)